### PR TITLE
Fixing grammar - refs https://www.transifex.com/projects/p/owncloud/t…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…ranslate/#en_GB/lib/50330746

easy one ....

@nickvergessen @MorrisJobke @th3fallen THX

### CODE REVIEW

**Comment by MorrisJobke** [comment]
:+1:

**Comment by th3fallen** [comment]
:+1:

File: lib/private/encryption/manager.php [hunk 1]
```diff
@@ -151,8 +151,8 @@ public function getEncryptionModule($moduleId = '') {
 			if (isset($this->encryptionModules[$moduleId])) {
 				return call_user_func($this->encryptionModules[$moduleId]['callback']);
 			} else {
-				$message = "Module with id: $moduleId does not exists.";
-				$hint = $this->l->t('Module with id: %s does not exists. Please enable it in your apps settings or contact your administrator.', [$moduleId]);
+				$message = "Module with id: $moduleId does not exist.";
```
Comment (LukasReschke) [inline lib/private/encryption/manager.php]: :boom: 

https://ci.owncloud.org/job/server-master-linux/database=sqlite,label=SLAVE/1994/testReport/junit/(root)/Test_Encryption_ManagerTest/testGetEncryptionModuleUnknown/

Comment (LukasReschke) [inline lib/private/encryption/manager.php]:
```
Test\Encryption\ManagerTest::testGetEncryptionModuleUnknown
Failed asserting that exception message 'Module with id: unknown does not exist.' contains 'Module with id: unknown does not exists.'.
```

Comment (LukasReschke) [inline lib/private/encryption/manager.php]: https://github.com/owncloud/core/pull/16730 will fix this

**Pull request merged**